### PR TITLE
[LG-4544] Add article about handling partner support tickets

### PR DESCRIPTION
**Pull request by orenyk**
Resolves LG-4544

### CODE REVIEW

File: _articles/partnerships-ticket-handling.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+description: "How to handle and track support requests from partners"
+category: Partnerships
+---
+
+# Partner support ticket handling
+
+## Overview
+
+This is a quick reference for how to handle support requests from partner. These
+can include questions on Slack, emails to partners@login.gov, or action items
```
Comment (zachmargolis) [inline _articles/partnerships-ticket-handling.md:13]: sometimes to minimize spam we try not to add email addresses to the handbook, could abbreviate this as `partners@` if we wanted? ditto for other parts of the file

**Review by zachmargolis — APPROVED** [review]
LGTM

File: _articles/partnerships-ticket-handling.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
+---
+title: "Partner Support Ticket Handling"
+layout: article
+description: "How to handle and track support requests from partners"
+category: Partnerships
+---
+
+# Partner support ticket handling
```
Comment (achapm) [inline _articles/partnerships-ticket-handling.md:8]: Looks like this displays as a duplicate title. See here:
https://federalist-7d0b2b76-42fc-4df1-9825-8c3cd77e0a15.app.cloud.gov/preview/18f/identity-handbook/oyk-partner-support-ticketing/articles/partnerships-ticket-handling.html
```suggestion
```

File: _articles/partnerships-ticket-handling.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+## Overview
+
+This is a quick reference for how to handle support requests from partner. These
+can include questions on Slack, emails to partners@login.gov, or action items
+from meetings with partners.
+
+For the purposes of this article, we define a **partner support ticket** as _a
+request from a partner that requires immediate action from the Partnerships
```
Comment (achapm) [inline _articles/partnerships-ticket-handling.md:17]:
```suggestion
request from a partner that requires action from the Partnerships
```
I would consider removing the word immediate, unless you want to specify a response window.

**Review by achapm — APPROVED** [review]
One suggestion above, otherwise LGTM.

File: _articles/partnerships-ticket-handling.md (hunk 3 above)
Comment (orenyk) [reply _articles/partnerships-ticket-handling.md:17]: sounds good!

File: _articles/partnerships-ticket-handling.md (hunk 2 above)
Comment (orenyk) [reply _articles/partnerships-ticket-handling.md:8]: Good catch!

File: _articles/partnerships-ticket-handling.md (hunk 1 above)
Comment (orenyk) [reply _articles/partnerships-ticket-handling.md]: sure thing!

**Pull request merged**